# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/xf86-video-xgi/hotfix.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- xf86-video-xgi-1.6.1/src/xgi_driver.c.vanilla	2022-08-25 10:51:28.536258244 +0200
+++ xf86-video-xgi-1.6.1/src/xgi_driver.c	2022-08-25 10:53:23.188251411 +0200
@@ -4500,7 +4500,7 @@
     XGIPtr pXGI = XGIPTR(pScrn);
 
     pScreen->BlockHandler = pXGI->BlockHandler;
-    (*pScreen->BlockHandler) (BLOCKHANDLER_ARGS);
+    //(*pScreen->BlockHandler) (BLOCKHANDLER_ARGS);
     pScreen->BlockHandler = XGIBlockHandler;
 
     if (pXGI->VideoTimerCallback) {
